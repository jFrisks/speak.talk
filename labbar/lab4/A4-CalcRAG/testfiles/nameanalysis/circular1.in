let a =
    let
        b = 2.0
    in
        a * b
    end
in
    a * 3.0
end