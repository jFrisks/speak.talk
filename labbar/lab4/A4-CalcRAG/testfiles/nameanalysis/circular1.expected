Error at line 5: the definition of symbol 'a' is circular